### ISSUE TITLE
Failed to run post update method

### ISSUE THREAD

**Issue by univeous** [imjp94/gd-plug#13]
```
SCRIPT ERROR: Invalid get index 'do' (on base: 'Dictionary').
          at: install (res://addons/gd-plug/plug.gd:433)
```

**Comment by univeous** [comment]
Actually #14 will not fix this, it's just a typo fix in README. 
I have no idea what does `plugin.do` means here so I have no clue how to fix it 🥲
https://github.com/imjp94/gd-plug/blob/cab046679de787f30ece8915a913a38bd0fc44e1/addons/gd-plug/plug.gd#LL433C77-L433C77

**Comment by imjp94** [comment]
I have not idea what it is XD, but I just fixed it in #16, should works fine now.
Try executing `upgrade` command, it should pull the latest update from master branch
